src/App.tsx

```diff
@@ -3,33 +3,50 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
 
   render() {
+    const { query } = this.state;
+
+    const visibleMovies = moviesFromServer.filter(movie => {
+      const title = movie.title.toLowerCase();
+      const description = movie.description.toLowerCase();
+
+      return title.includes(query) || description.includes(query);
+    });
+
     return (
       <div className="page">
         <div className="page-content">
           <div className="box">
             <div className="field">
               <label htmlFor="search-query" className="label">
                 Search movie
+                <div className="control">
+                  <input
+                    type="text"
+                    id="search-query"
+                    name="query"
+                    className="input"
+                    placeholder="Type search word"
+                    value={query}
+                    onChange={(event) => {
+                      this.setState({ query: event.target.value.toLowerCase() });
```

Comment:
smart decision. but it might confuse some people that they cannot enter capital letters. you can move this logic to the filter method in visibleMovies